### DISCUSSION TITLE
JWT Authentication with AAD

### DISCUSSION THREAD

**Discussion by cmjdev** [prest/prest#460] [API Server] [answered]
We are wanting to use pREST to build quick internal tools at our company. We use Azure Active Directory and would like to set permissions for endpoints with pREST using AAD Roles via JWT.  Is this a possibility in current state?

**Comment by avelino** [comment, marked as answer]
Today we don't support AAD when using the pREST binary (It is an interesting resource that can add value to the issue we are implementing https://github.com/prest/prest/issues/276), but you can use pREST as a framework and create your authentication.

More about pREST framework: https://docs.postgres.rest/prest-as-web-framework/
Example: https://github.com/prest/prest-example/blob/master/custom-middlewares/main.go#L24
